src/app/zeko/da-layer/contracts/DataAvailability.sol

```diff
@@ -11,9 +11,9 @@ contract DataAvailability is MinaMultisig {
     using FieldBytes for bytes;
     using Endianness for bytes32;
 
-    event BatchPosted(bytes32 indexed id);
+    event BatchPosted(uint256 indexed location);
```

Comment:
Why change this?

Reply:
It's just a rename, before I've called it batchId, not it's location which I think is better.